blog/2020-10-23-supervisor-update.md

```diff
@@ -0,0 +1,90 @@
+---
+author: Joakim Sørensen
+authorURL: https://github.com/ludeeus
+authorTwitter: ludeeus
+title: "Upcoming changes to add-ons"
+---
+
+## GitHub Action
+
+You can now use our [builder][marketplace] as a [GitHub action][github_action]! :tada:
+
+This is already in use in our [hassio-addons repository][addons], you can see an example on how we implemented it [here][builder-action].
+
+It can be used both to ensure that the add-on will still build with changes made to your repository, and to publish the images as part of a release workflow. How to use the action is documented in the [builder repository][builder-action].
```

Comment:
```suggestion
It can be used to ensure that the add-on will still build with changes made to your repository and publish the images as part of a release workflow. How to use the action is documented in the [builder repository][builder-action].
```